Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -68,6 +68,19 @@ public LayoutAnchor Anchor
             set { anchor = value; }
         }
 
+        [SerializeField, Tooltip("Whether or not to use the legacy anchor layout")]
+        private bool useLegacyAnchor = true;
+
+        /// <summary>
+        /// Where the grid is anchored relative to local origin
+        /// </summary>
+        public bool UseLegacyAnchor
```

Comment:
Possibly a nit, but I wonder if there's a better name we can use here. If I'm just looking through the properties, I'm not sure what "legacy" would be referring to. Possibly something that can be solved via documentation though, or updating the summary tag and tooltip to more explicitly describe what happens when i check / uncheck this.

Reply:
Maybe "Use MRTK v2.3 anchoring"?

Reply:
I'll change it to "anchor along axis" since that succinctly describes its behavior. When the anchor position is set to "Middle Center" I'll remove the checkbox from the editor to avoid any confusion

Reply:
>Use MRTK v2.3 anchoring

I'm not sure about that one either, since what if 2.4 is my first experience with MRTK or this component?
I think "anchor along axis" is good